routes/freelance/router.js

```diff
@@ -240,6 +240,114 @@ router.put('/:freelanceid/availability', function(req, res, next) {
   }
 });
 
+// PUT freelance/:freelanceid/edit/
+router.put('/:freelanceid/edit', function(req, res, next) {
+  // TODO must Add freelance in the tags reference.
```

Comment:
As said in Slack, just happened to me that a `DELETE /freelance/:id` just responded with `400` because there was a problem finding a `tag`, however then after reloading the freelancer was actually removed...

I remember that I had edited the tags, so probably I also removed one, or some other kind of problem that could be related to this.